apps/kyb-app/src/pages/CollectionFlow/versions/v2/CollectionFlowV2.tsx

```
@@ -88,338 +89,348 @@ export const CollectionFlowV2 = withSessionProtected(() => {
-                                        ) : (
-                                          <span className="italic">
-                                            Page not found - state: {state}
-                                          </span>
+                                      <div className="min-h-0 flex-1 pb-10">
+                                        {isLogoLoaded ? <StepperUI /> : null}
+                                      </div>
+                                      <div>
+                                        {themeDefinition.settings?.contactInformation ? (
+                                          <div
+                                            className="text-sm"
+                                            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)